4-forms-validations/form.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial=scale=1.0">
+        <meta name="description" content="AUTHORIZATION FORM">
+        <meta name="keywords" content="FORM, LEARNING, PROGRAMING">
+        <meta name="author" content="Jaime Moreno"> 
```

Comment:
Very nice to see these tags, it's very common to forget it.

Reply:
You can add in your keywords the topics that we are touching: `html`, `accessibility` `semantic` `forms`